pkg/controller/volume/persistentvolume/scheduler_assume_cache.go

```diff
@@ -378,7 +378,7 @@ type PVCAssumeCache interface {
 	AssumeCache
 
 	// GetPVC returns the PVC from the cache with the same
-	// namespace and the same name of the specified pod.
+	// namespace and the same name of the specified PVC.
```

Comment:
It's indeed a typo. But I'm wondering if the whole statement is not that appropriate here.
I think the comment here should be originally copied from [scheduler cache ](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/internal/cache/interface.go#L84), which takes an object but not a key as input.

Maybe we should make it "GetPVC returns the PVC from the cache with given pvcKey" here.